src/experimental/patterns/ec2-app.ts

```diff
@@ -261,7 +276,7 @@ export interface GuEc2AppExperimentalProps extends Omit<GuEc2AppProps, "updatePo
  * If you need to use this class for a service other than MAPI, please speak to DevX about your use-case first.
  */
 export class GuRollingUpdatePolicyExperimental {
-  static getPolicy(maximumInstances: number, minimumInstances?: number) {
+  static getPolicy(maximumInstances: number, minimumInstances?: number, slowStartDuration?: Duration) {
```

Comment:
To get this working correctly for MAPI we'll need to pass the slow start duration in here...